marlowe-playground-client/src/GistButtons.purs

```diff
@@ -20,13 +20,16 @@ authButton intendedAction state =
     authStatus = state ^. (_authStatus <<< to (map (view authStatusAuthRole)))
   in
     case authStatus of
+      -- TODO: Check how it looks visually and rethink if it should have a CTA to retry
       Failure _ ->
         button
           [ idPublishGist
           ]
           [ text "Failed to login" ]
       Success Anonymous ->
-        div [ class_ (ClassName "auth-button-container") ]
+        -- TODO: Validate this dialog with the designer, see if we should add a title
+        --       based on the intendedAction
+        div [ classes [ ClassName "modal-content", ClassName "auth-button-container" ] ]
```

Comment:
I'd favour extracting any class names you use more than once to a top-level definition. (If they're one-off, I think it's okay to define them inline.)

Reply:
done